src/components/activity-list/RecyclerActivityList.js

```diff
@@ -115,7 +115,7 @@ export default class RecyclerActivityList extends PureComponent {
         } else if (type === ViewTypes.HEADER) {
           dim.height = 39;
         } else {
-          dim.height = props.isLoading
+          dim.height = this.props.isLoading
             ? deviceUtils.dimensions.height
             : props.addCashAvailable
```

Comment:
Can u also change this one to `this.props.addCashAvailable`

Reply:
that check seems to be working, that's why I left it

still worth changing?

Reply:
The reason that one is working is because we’re not changing the value on runtime (like we do with isLoading).

I still think it’s worth changing because if in the future we decide to turn off addCash on runtime for some reason,  it won’t get the updated value

Reply:
cool, makes sense